fix for http4s fetch() deprecation warnings

jacum opened the pull request:
http4s 0.21.5+ deprecates 'fetch' operation.
This PR should address it.

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

kelnos commented:
Please rebase off master; it looks like somehow one of my commits has ended up in your PR, rewritten somehow.  Also, tests are failing?

Will this change affect compatibility?  That is, are the new method names available in the older versions of http4s that guardrail supports?

jacum commented:
will reopen new one

jacum commented:
Hi Brian,

PR re-submitted as
https://github.com/twilio/guardrail/pull/769 <https://github.com/twilio/guardrail/pull/769>

Also added the compatibility notes.


> On 2 Sep 2020, at 20:45, Brian Tarricone <notifications@github.com> wrote:
> 
> 
> Please rebase off master; it looks like somehow one of my commits has ended up in your PR, rewritten somehow. Also, tests are failing?
> 
> Will this change affect compatibility? That is, are the new method names available in the older versions of http4s that guardrail supports?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/twilio/guardrail/pull/764#issuecomment-685927508>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAD24OTUQW2LU6SELAGMSC3SD2HEHANCNFSM4QSWHLTQ>.
>
